code/__DEFINES/station.dm

```diff
@@ -2,8 +2,15 @@
 #define STATION_TRAIT_NEUTRAL 2
 #define STATION_TRAIT_NEGATIVE 3
 
-
+/// For traits that shouldn't be selected, like abstract types (wow)
 #define STATION_TRAIT_ABSTRACT (1<<0)
+/// Only run on planet stations
+#define STATION_TRAIT_PLANETARY (1<<1)
+/// Only run on space stations
+#define STATION_TRAIT_SPACE_BOUND (1<<2)
+
+/// Not restricted by space or planet, can always just happen
+#define STATION_TRAIT_UNRESTRICED STATION_TRAIT_PLANETARY | STATION_TRAIT_SPACE_BOUND
```

Comment:
```suggestion
#define STATION_TRAIT_UNRESTRICTED STATION_TRAIT_PLANETARY | STATION_TRAIT_SPACE_BOUND
```
Although maybe it should be `TRAIT_MAP_UNRESTRICTED` too just for increased specificity.
We also maybe could have avoided replacing all of those NONEs by inverting the trait behaviour to "not on a planet" but that would result in a pretty ugly trait name and I think positive traits are probably a better pattern.

Reply:
Welp that's awkward. Fixed now